demos/animation/anim.go

```diff
@@ -96,6 +97,7 @@ func main() {
 		Title:  "Animation Demo",
 		Width:  1024,
 		Height: 640,
+		DefaultCloseAction,
```

Comment:
Also, having to set `true` to everything, when it's default, seems a bit redundant. Can you rename it such that the value `false` is the default? (without changing behavior? )